### ISSUE TITLE
Discuss and evaluate converting an existing project to support WTS>Add

### ISSUE THREAD

**Issue by mrlacey** [Microsoft/WindowsTemplateStudio#663]
Creating this issue to separate the discussion from other issue (X-Ref #623)

@crutkas said

> Should we include an issue to have right click add for non-WTS issues.

@ralarcon said

> I think so, anyway we need to evalutate the risk of manipulating the developer project.

@sibille said

> I had an approach for this in the poc_rightclick branch.
If the project is non-WTS you can choose the projectType and framework and then an "incrementproject" template was generated that contains all the things that would have been added if you created your project with wts without creating a new project (ActivationSerivce, NavigationService).
I had an issue with adding nuget references in the .csproj file as we use an anchor () for this.

**Comment by mrlacey** [comment]
Further discussion has been had on this topic

Add WTS Scaffolding (manifest entries) from new Right-click option: 'Enable WTS"'
* Prompt for project type/framework, add to manifest, Then show Add options
* Infer project type/framework and add to manifest. Then show Add options

Preferred solution: prompt but try and infer to preselect the appropriate options.
Review this approach based on real world usage/feedback.

Imporve the change summary (step 2 in add wizard)
- Only include the unchanged files when in debug (as this is of no real value)
- Add checkbox options to allow choosing which files to integrate in the project. (If not a geenrated project there is a lot that coudl go wrong here and it may be prefereable to not try and merge some files.)
- Make wanring text crisper/clearer
- Review all post action file descriptions (Ensure all present and correct)
- Change message of page already added to: a page named Blank.... instead of "Blank is already added"

**Comment by crutkas** [comment]
good point bringing this item back up the chain, i do believe this is a critical item we should investigate as well
